### PR TITLE
add official Django 5.1 support

### PR DESCRIPTION
**Pull request by GeyseR**
adds explicit support for Django 5.1

### CODE REVIEW

**Review by sjdemartini — APPROVED** [review]
Thank you, LGTM!

File: tox.ini [hunk 1]
```diff
@@ -2,8 +2,8 @@
 envlist =
     py{38,39,310}-django32
     py{38,39}-django42
-    py{310,311}-django{42,50,main}
-    py312-django{42,50,main}
+    py{310,311}-django{42,50,51,main}
+    py312-django{42,50,51,main}
```
Comment (kiendang) [inline tox.ini:6]: I think we can simplify this to

```suggestion
    py{310,311,312}-django{42,50,51,main}
```

Comment (GeyseR) [reply tox.ini]: good point, done!

**Review by kiendang — APPROVED** [review]
Thank you @GeyseR

**Pull request merged**